_terms/legal-privacy-policy.md

```diff
@@ -46,26 +47,30 @@ SixSq Info: info@sixsq.com
 [SixSq](https://sixsq.com "SixSq's Homepage")
 
 
-2\. What personal data we collect from you?
+2\. What personal data do we collect from you?
 ---  
 
-During the process of registration for our services and products, we may collect personal and unique information, which can be used to identify you, such as your email address, name, organisation, and any other information that you might willingly share when you authenticate via a social provider or an identity federation.
+During the process of registration for our services and products, we may collect personal and unique information which can be used to identify you, such as your:
 
-With respect to the Federated Identity Portal, the following data, if available, (with respective attribute names in SAML:OIDC) are required and retrieved from the persons’ Home Organisation:
+- email address
+- name
+- organisation
 
-- email (email:mail)
-- user identifier (eduPersonPrincipalName:preferred_username)
-- type of affiliation (eduPersonAffiliation)
-- person’s affiliation within a particular security domain (eduPersonScopedAffiliation)
-- home organization type (schacHomeOrganizationType)
-- person’s home organization using the domain name of the organization (schacHomeOrganization)
-- persistent, non-reassigned, opaque identifier for a principal (eduPersonTargetedID)
-- the names of an object (commonName)
-- the name(s) that should appear in white-pages-like applications (displayName:name)
-- name strings that are the part of a person’s name that is not their surname (givenName)
-- name strings for the family names of a person (sn)
-- long-lived, non re-assignable, omnidirectional identifier (eduPersonUniqueId:sub)
-- URI (either URL or URN) that indicates a set of rights to specific resources (eduPersonEntitlement)
+and any other information that you might willingly share when you authenticate via a social media provider or identity provider.
+
+We may also ask you to provide further information in order to invoice you electronically for our services. SixSq uses the [Stripe](https://stripe.com) payment service for this. This data is not held by SixSq, but transferred to Stripe. This information can include:
+
+- email
+- name
+- credit card details
+
+Our services may also use the [Intercom](https://www.intercom.com) online service to provide customer support and in-app chat and communication. Intercom receives the following information from us:
+
+- email
+- status of trial
+- key events (e.g. first deployment, first application creation, first NuvlaBox registration)
+
```

Comment:
I think Intercom collects much more than this. I'm sure it collects location for example...which I'm not sure how it works since we are the ones relaying the user registration...so are we the ones inferring the user location? if so, then that needs to go into our personal data collection

Reply:
No we're not inferring this. I don't know how it does that. IP reverse?

Reply:
meaning that we forward the requester's IP to them?